scripts/cmake/test/RunBenchmark.cmake

```diff
@@ -1,4 +1,4 @@
-set(BENCHMARK_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/benchmarks/${benchmarkDir})
+set(BENCHMARK_OUTPUT_DIRECTORY ${BENCHMARK_DIR}/${benchmarkDir})
```

Comment:
The intention here is to have benchmark output files just in the build-director to keep the source dir in a clean state. This is especially helpful in Jenkins to have a build and benchmark run in one enclosed directory.

So I suggest to either revert this change or introduce another variable (e.g. `BENCHMARK_OUTPUT_DIR`).